packages/calcite-components/src/components/button/button.scss

```diff
@@ -228,271 +253,288 @@
 :host([appearance]) {
   button,
   a {
-    @apply border-color-transparent
-      border
-      border-solid;
+    background-color: var(--calcite-button-background-color);
+    color: var(--calcite-button-text-color);
+    border: var(--calcite-border-width-sm) solid var(--calcite-button-border-color);
+    box-shadow: var(--calcite-button-box-shadow);
```

Comment:
Based on this usage, `--calcite-button-box-shadow` defines the entire box shadow, not just the color as the description above indicates.  Also, should we be setting both `border` and `box-shadow` together?  Seems like we should stick with just one for simplicity.

Reply:
Yeah this one was tricky, I asked Ali about it and they asked Aaron about it, but there was no precedence set for how we wanted to handle that yet so for now they said that there should just be one token called `--calcite-button-box-shadow` and for now just to apply the existing values that were there before to it.

Reply:
This makes sense IMO as our box-shadow tokens set all portions of a box-shadow rule. Border-color should be separate as it is here.

Reply:
We should update the @prop doc above then to say "Specifies the box shadow of the component" or something to that effect.  I also again think we should revisit border vs. box-shadow.  I know we use box-shadow to create border effects instead of CSS `border` in other places because of all the extra control you get with it, but IMO we should stick to one vs. the other unless there's a specific use case we're trying to solve for that necessitates both.

Also, not sure if naming the css property to include the words "box shadow" makes sense if one of our goals is not to leak implementation details or not.  I don't have a hard opinion on that since this is would be applying a web-platform variable and we should use those terms in our documentation since that's the target platform for these variables in this case, but thought that would be a good point to bring up too.

@macandcheese @ashetland @SkyeSeitz: is there a use case for supporting both a `border` _and_ a `box-shadow` on button?

Reply:
I take that back what I said about border vs box-shadow.  I think its valid to include both of them just because of the nature of what box-shadow allows you to do.  I'm noticing Adam's work on Card is similar to what someone might want to do with button.